packages/frontend/src/services/indexer.js

```diff
@@ -1,17 +1,33 @@
 import { INDEXER_SERVICE_URL } from '../config';
 import sendJson from '../tmp_fetch_send_json';
+import { buildUrlOptParam } from '../utils/url';
+import IndexerCache from './lib/indexerCache';
+
+const cache = new IndexerCache();
+cache.open();
+
+function cachedRequest (accountId, kind) {
+    const url = `${INDEXER_SERVICE_URL}/account/${accountId}/${kind}`;
```

Comment:
I think its better to distinct logic of building access URL to access API and Cache ids (kind)